### DISCUSSION TITLE
Recieved multiple suspicious emails

### DISCUSSION THREAD

**Discussion by FelixSiegel** [community/community#135571] [GitHub Education]
### Hi everyone,

Hey,
I've been receiving several emails for a while now that seem very strange to me.
The title and content are filled with lots of emojis and "click me and see what happens" buttons in a very dubious way.
It is usually said that this comes from hackclub and links in the email also lead there. Since hackclub is actually something official and the sender is also "githubeducation@github.com", I'm currently not sure whether this is really genuine or a scam.

I found these two links from github myself, where no such email type is mentioned:

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/types-of-emails-github-sends

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/managing-marketing-emails-from-github

And I also looked in my account, but there is no supsciption thing like that there:

![image](https://github.com/user-attachments/assets/ecd742be-f5e3-418b-80ae-27ea7ad32aa3)

---

Here is a screenshots of the emails i recieved and ment:

![image](https://github.com/user-attachments/assets/c6a85475-60f4-42db-8b9b-cfdf0075a904)

The latest email contains content like the following:

![image](https://github.com/user-attachments/assets/d703f0ec-d682-4746-9ecd-672386bebb92)

![image](https://github.com/user-attachments/assets/9e778c99-c7c8-4e22-af3e-3c6a003a078d)

![image](https://github.com/user-attachments/assets/b630bec8-8ea7-4fbc-bcd1-aeffc48ea60d)

I am particularly surprised by such things:

![image](https://github.com/user-attachments/assets/284af31f-3240-4da5-8df3-4a0070517334)

![image](https://github.com/user-attachments/assets/df7e4ce4-6827-4c00-8a88-17667a9c8fb3)

If this is really an official email from Github, then what are those age tags along with heart emojis doing there? And why are there buttons I should click to see what they do? This seems really weird to me...

Please let me know, if this is real email or scam because I'm very confused right now.

Thanks.

**Comment by davevad93** [comment]
Hi @FelixSiegel ,

I've also received those emails because I'm subscribed to GitHub Education, to be honest I've never bothered to check it out since it seems to be content aimed at a teenage audience, although I think it's not scam as GitHub and Hack Club have a partnership as stated in [this post](https://x.com/GitHubEducation/status/1802745095285780595) from the official [GitHub Education](https://x.com/GitHubEducation) profile on Twitter/X. If you want to be 100% sure you should wait for an official response from someone from GitHub staff, but I personally believe that those emails are legit.
